src/mockDB/events.json

```diff
@@ -2,43 +2,51 @@
     { "id":"1",
         "eventName": "Hörnerfest",
         "eventLocation": "Brande-Hörnerkirchen",
-        "eventDate": "2020-06-25 - 2020-06-28"
+        "eventDate": "2020-06-25 - 2020-06-28",
```

Comment:
I recommend having `startDate` and `endDate` values in the future, and it's probably better to use numbers here as well. I know it's a pain in the ass sometimes, but try not to push it too far into the dev cycle :D
https://javascript.info/date

Oh and also you don't really need the `event` prefix here, I think it's clear that your data describes events :)

Reply:
I changed it to start and end date, because i'm using also startDate and endDate  later,  I just changed the prefix name on the other props